sdktests/php_eval.go

```diff
@@ -10,9 +10,9 @@ func doPHPEvalTests(t *ldtest.T) {
 	// how we set up the polling endpoints in the mock data source, and that is handled
 	// transparently by the logic in mockld/polling_service.go based on the fact that the
 	// "SDK kind" configured for test suite is PHP.
-	t.Run("parameterized", runParameterizedServerSideEvalTests)
```

Comment:
Do these deserve TODOs? I feel like it could get lost.

Reply:
I can actually add this back in, so we are good there.